R/helpers.R

```diff
@@ -520,14 +520,18 @@ myBarplot <- function(x, description, xaxisround=0)
 #'
 #' @param target (`numeric`)\cr target values.
 #' @param current (`numeric`)\cr current values.
-#' @param tolerance (`number`) differences smaller than this are not reported.
+#' @param tolerance (`number`) relative differences smaller than this are not reported.
 #' @return `TRUE` when `target` and `current` do not differ
 #'   up to desired tolerance and without looking at names or other attributes,
 #'   `FALSE` otherwise.
 #'
 h_all_equivalent <- function(target, 
                              current, 
                              tolerance = 1e-10) {
+  assert_numeric(target)
+  assert_numeric(current)
+  assert_number(tolerance)
```

Comment:
note that I did not assert this argument before because it can be done by `all.equal` but it is ok to do it.